hack/get-project-maintainers.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+# Copyright 2021 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+REPO_ROOT=$(dirname "${BASH_SOURCE[0]}")/.. 
+cd "${REPO_ROOT}"
+
+YQ=$(realpath hack/tools/bin/yq)
+make "${YQ}" &>/dev/null
```

Comment:
Calling `make` from the script that was invoked to populate a Makefile variable... 🤯

Reply:
I could have also put it as a make target dependency but I figured the script should work even when it's not called from that make target so it's better to install the dependency directly in the script